Jenkinsfile

```diff
@@ -19,7 +19,7 @@ pipeline {
 					sh """
 					mvn clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
 						-Pbuild-individual-bundles -Pbree-libs -Papi-check \
-						-Dcompare-version-with-baselines.skip=false \
+						-Dcompare-version-with-baselines.skip=true \
```

Comment:
This change will make verification build not fail when version bumps are needed thus it sholdn't happen.

Reply:
Yes I was trying out this as I still had the " [ERROR] META-INF/ECLIPSE_.SF: present in baseline only". The intention was to check for other issues and then revert in the next commit  in this PR( as already done)